stablehlo/tests/CheckOps.cpp

```diff
@@ -97,34 +109,45 @@ static llvm::ErrorOr<std::string> getSerializedTensorPath(StringRef probeId,
   if (!metadataFile.is_open()) return llvm::errc::io_error;
 
   std::string line;
+  std::vector<std::string> fields(3);
 
   for (uint32_t match = 0; metadataFile >> line && match <= iteration;
        ++match) {
     auto pos = line.find(probeId);
 
-    if (pos != std::string::npos && match == iteration)
-      return line.substr(pos + probeId.size() + 1);
+    if (pos != std::string::npos && match == iteration) {
+      // Parse the record in the form of: probe_id,mlir_type,serialized_path
+      splitString(line, ',', fields.begin());
+      return std::make_pair(/*type=*/fields[1], /*path=*/fields[2]);
+    }
   }
 
   return llvm::errc::bad_address;
 }
 
 llvm::Error evalExpectSerializedEqOp(const Tensor &expected, StringRef probeId,
                                      StringRef probeDir, uint32_t iteration) {
-  auto serializedFilePathOrError =
-      getSerializedTensorPath(probeId, probeDir, iteration);
+  auto serializedMetadataOrError =
```

Comment:
`serializedMetadata` - I didn't catch this last time, but we tend to avoid `OrFailure` / `OrError` names

Reply:
There's a couple of other `OrError` variable names in other files I introduced in a prior PR. I'll create a separate PR for those updates as well, and update the ones in the function already modified here in this PR.